modules/openapi-generator/src/main/resources/Java/libraries/microprofile-kumuluzee/pojo.mustache

```diff
@@ -0,0 +1,129 @@
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlType;
+import javax.xml.bind.annotation.XmlEnum;
+import javax.xml.bind.annotation.XmlEnumValue;
+import javax.json.bind.annotation.JsonbProperty;
+
+{{#withXml}}
+@XmlAccessorType(XmlAccessType.FIELD)
+{{#hasVars}} @XmlType(name = "{{classname}}", propOrder =
+    { {{#vars}}"{{name}}"{{^-last}}, {{/-last}}{{/vars}}
+}){{/hasVars}}
+{{^hasVars}}@XmlType(name = "{{classname}}"){{/hasVars}}
+{{^parent}}@XmlRootElement(name="{{classname}}"){{/parent}}
+{{/withXml}}
+{{#description}}
+/**
+  * {{{description}}}
+ **/
+{{/description}}
+public class {{classname}} {{#parent}}extends {{{parent}}}{{/parent}}{{#serializableModel}} implements Serializable{{/serializableModel}} {
```

Comment:
Also please add the option {{>additionalModelTypeAnnotations}} on the top of the public class {{ classname }} for user to add custom annotations
![image](https://user-images.githubusercontent.com/11788842/79985647-43a21500-84ee-11ea-944d-b3f262653132.png)